src/cpp/rtps/builtin/BuiltinProtocols.cpp

```diff
@@ -21,12 +21,12 @@
 #include <fastdds/rtps/common/Locator.h>
 
 #include <fastdds/rtps/builtin/discovery/participant/PDPSimple.h>
-// #include "./discovery/participant/PDPClient2.hpp"
-#include <fastdds/rtps/builtin/discovery/participant/PDPClient.h>
-#include "./discovery/participant/PDPServer2.hpp"
 #include <fastdds/rtps/builtin/discovery/endpoint/EDP.h>
 #include <fastdds/rtps/builtin/discovery/endpoint/EDPStatic.h>
 
+#include "./discovery/participant/PDPServer.hpp"
+#include "./discovery/participant/PDPClient.h"
```

Comment:
All `.h` and `.hpp` files are accessible in the sources. Please avoid all relative includes. As an example:
```suggestion
#include <rtps/builtin/discovery/participant/PDPServer.hpp>
#include <rtps/builtin/discovery/participant/PDPClient.h>
```